content/_data/changelogs/weekly.yml

```diff
@@ -7957,6 +7957,97 @@
   # pull: 4777 (PR title: Use the 'agent' term more often)
   # pull: 4778 (PR title: Fixed some style errors in less files to reduce output during build)
 
+- version: "2.242"
+  date: 2020-06-23
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4802
+    issue: 62622
+    authors:
+    - daniel-beck
+    message: |-
+      * Do not log exceptions when browsing to the plugin manager while private source plugins are installed (regression in 2.240).
+  - type: bug
+    category: bug
+    pull: 4813
+    issue: 61438
+    authors:
+    - rahulsom
+    references:
+      - issue: 61438
+      - url: https://github.com/stapler/stapler/releases/tag/stapler-parent-1.260
+        title: Stapler 1.260 changelog
+    message: |-
+      Fix <code>IllegalArgumentException</code> for Choice Parameter when saving job configuration.
+  - type: bug
+    category: bug
+    pull: 4813
+    issue: 61438
+    authors:
+    - rahulsom
+    references:
+      - pull: 4813
+      - url: https://github.com/stapler/stapler/releases/tag/stapler-parent-1.260
+        title: Stapler 1.260 changelog
+    message: |-
+      Correctly encode control and surrogate characters when exporting data to JSON.
+  - type: bug
+    category: bug
+    pull: 4801
+    issue: 62714
+    authors:
+    - clywyk
+    message: |-
+      Replace esclave with agent in French documentation.
+  - type: bug
+    category: localization
+    pull: 4809
+    authors:
+    - stikonas
+    message: |-
+      Fix a typo in Lithuanian translation.
+  - type: rfe
+    category: rfe
+    pull: 4814
+    authors:
```

Comment:
Add a reference to the dark theme plugin? https://github.com/jenkinsci/dark-theme-plugin